Add alt text to Dask docs (#1)
scharlottej13/dask: docs/source/shared.rst

The schedulers that depend on some sort of asynchronous pool have costs of a few
milliseconds and the single threaded schedulers have costs of a few microseconds.

.. figure:: images/scaling-nodes.png

>    :alt: Duration in seconds on the y-axis versus number of edges per task on the x-axis. 

Author: ```suggestion
   :alt: Duration in seconds on the y-axis versus number of edges per task on the x-axis. The cost per edge decreases up until about 10 edges, after which the cost plateaus for both the threaded and async schedulers, with the async scheduler being consistenly faster.
```

>    :alt: Duration in seconds on the y-axis versus number of edges per task on the x-axis. 

Author: ```suggestion
   :alt: Duration in seconds on the y-axis versus number of edges per task on the x-axis. The time to schedule the entire graph is constant initially, followed by a linear increase after roughly 500 tasks for multiprocessing and threaded schedulers and 10 tasks for async and core schedulers. The inverse is true for the cost per task, with a linear cost decrease, followed by more or less constant cost.
```

>    :alt: Duration in seconds on the y-axis versus number of edges per task on the x-axis. The time to schedule the entire graph is constant initially, followed by a linear increase after roughly 500 tasks for multiprocessing and threaded schedulers and 10 tasks for async and core schedulers. The inverse is true for the cost per task, with a linear cost decrease, followed by more or less constant cost.

Reviewer: ```suggestion
   :alt: Graph depicting how well Dask scales with the number of nodes in the task graph. Graph shows the duration in seconds on the y-axis versus number of edges per task on the x-axis. The time to schedule the entire graph is constant initially, followed by a linear increase after roughly 500 tasks for multiprocessing and threaded schedulers and 10 tasks for async and core schedulers. The inverse is true for the cost per task, with a linear cost decrease, followed by more or less constant cost.
```

Good to start with a sentence describing the whole image, before diving in to the details.

communication; they have been removed from the plot.

> .. image:: images/scaling-edges.png

Author: ```suggestion
.. figure:: images/scaling-edges.png
```

>    :alt:

Reviewer 2: I'm working on this

```suggestion
   :alt: As the number of edges increase from 0 to 100 the time to schedule the entire graph using the threaded scheduler goes from 2 to 8 seconds and using the async scheduler goes from 0 to 3 seconds.
```

>    :alt:

Reviewer 3: ```suggestion
   :alt: Two graph panels. Left: Cost of entire graph, Duration versus Number of edges per task. Dask.async and dask.threaded curves both increase linearly, with the latter consistently higher by at least a factor of 3. Right: Cost per edge, with the same axes. Both curves start from a high value for small number of edges, but reach an asymptote where dask.threaded is a factor of a few higher than dask.async .
```

>    :alt: As the number of edges increase from 0 to 100 the time to schedule the entire graph using the threaded scheduler goes from 2 to 8 seconds and using the async scheduler goes from 0 to 3 seconds.

Author: ```suggestion
   :alt: Duration in seconds on the y-axis versus number of edges per task on the x-axis. As the number of edges increases from 0 to 100, the time to schedule the entire graph using the threaded scheduler goes from 2 to 8 seconds whereas using the async scheduler goes from 0 to 3 seconds.
```

> 

Author: ```suggestion
:caption: Scheduling overhead cost of the entire graph (left) vs. cost per edge (right) as the number of tasks increase.
```

are good at workflows with non-trivial cross-task
communication; they have been removed from the plot.

.. figure:: images/scaling-edges.png

>    :alt: Duration in seconds on the y-axis versus number of edges per task on the x-axis. As the number of edges increases from 0 to 100, the time to schedule the entire graph using the threaded scheduler goes from 2 to 8 seconds whereas using the async scheduler goes from 0 to 3 seconds. The cost per edge decreases up until about 10 edges, after which the cost plateaus for both the threaded and async schedulers, with the async scheduler being consistenly faster.

Reviewer: ```suggestion
   :alt: Graph depicting how well Dask scales with the number of edges in the task graph. Graph shows the duration in seconds on the y-axis versus number of edges per task on the x-axis. As the number of edges increases from 0 to 100, the time to schedule the entire graph using the threaded scheduler goes from 2 to 8 seconds whereas using the async scheduler goes from 0 to 3 seconds. The cost per edge decreases up until about 10 edges, after which the cost plateaus for both the threaded and async schedulers, with the async scheduler being consistenly faster.
```

Reviewer: Same as above, it's good to start with the big picture before diving into the nitty gritty details.
